Dev ieduplicates sj

luizaandrade opened the pull request:
These are the changes we have agreed to. I'll do the issues still being discussed in a different branch, so we can decide whether to merge them or not.

kbjarkefur commented:
@luizaandrade , I do not have time to check right now, can look again later or tomorrow. But my daily file still only have a date stamp, not a time stamp as discussed in #95

kbjarkefur commented:
Apart from the time stamp not working and the comments I have made in issues related to ieduplicates, and  the comment I made [here](https://github.com/worldbank/iefieldkit/commit/fdc3fae9691163ec9700ac47f493010ef280a4a5), it all seems to be working correctly.

luizaandrade commented:
Ok, I have:
- Changed to option name back to `force`. Since I'm the swing vote, I decide.
- Added an error message if `cd` is used that links to the wiki and explains why it should not be used
- Checked that the timestamp is working

kbjarkefur commented:
Thats neat that it only shows the time stamp if two different files are created the same day. 

Everything looks good to me now! I think this is ready to be merged!
